build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+import configurations.Languages.attachLocalRepositories
+
+import dependencies.Libraries.ANDROID_BUILD_TOOLS
+
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+    }
+
+    dependencies {
+        classpath(ANDROID_BUILD_TOOLS)
+    }
+}
+
+allprojects {
+    group = "org.worldcubeassociation.tnoodle-samples"
+    version = "0.14.0"
```

Comment:
Should we restart this version number so it's clear that it's unrelated to TNoodle core?

Reply:
Fixed!